server/server.js

```diff
@@ -2,44 +2,21 @@ require('dotenv').config();
 const express = require('express');
 const bodyParser = require('body-parser');
 const morgan = require('morgan');
-const session = require('express-session');
 const app = express();
 const PORT = process.env.PORT || 8080;
-const dbConnection = require('./database');
-const MongoStore = require('connect-mongo')(session);
-const passport = require('./passport');
+require('./database');
```

Comment:
@LGG233 We no longer need to create sessions in our own database; the access tokens handle this for us. We don't require the connection object from `./database` explicity, however we do need the connection logic in there to execute to create the database connection before we do anything, hence the unassigned `require('./database')`.